Source/PercentageGradient.swift

```diff
@@ -7,18 +7,53 @@ public enum GradientDirection {
 
 }
 
+public struct GradientPoints {
+    
+    public var start: CGPoint
+    public var end: CGPoint
+    
+}
+
+public enum GradientAngle {
+    
+    case custom(start: CGPoint, end: CGPoint)
+    case defaultAngle
+    case bottomLeftToTopRight
+    case topLeftToBottomRight
+    case horizontal
+    case vertical
+    
+    func gradientPoints() -> GradientPoints {
```

Comment:
I missed this my first time through, but can we make this a computed property instead of a function, to match the other properties we have?

Probably personal preference, but I like `self.gradient.angle.gradientPoints.start` more than `self.gradient.angle.gradientPoints().start` when compared to the rest of the library.

Reply:
I actually wanted to make this change and forgot, I like that call site much better!  Great suggestion, thanks for catching.